Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateMediaStreamAVFObjC.h

```diff
@@ -135,8 +135,8 @@ class MediaPlayerPrivateMediaStreamAVFObjC final
 
     bool seeking() const override { return false; }
 
-    std::unique_ptr<PlatformTimeRanges> seekable() const override;
-    std::unique_ptr<PlatformTimeRanges> buffered() const override;
+    const PlatformTimeRanges& seekable() const override;
+    const PlatformTimeRanges& buffered() const override;
```

Comment:
final?

Reply:
that's existing and all thorough this class, will do so in a separate patch